course/src/main/java/org/openedx/course/presentation/outline/CourseOutlineFragment.kt

```diff
@@ -392,11 +389,19 @@ internal fun CourseOutlineScreen(
                                         Box(listPadding) {
                                             if (windowSize.isTablet) {
                                                 ResumeCourseTablet(
+                                                    modifier = Modifier.padding(
+                                                        top = 16.dp,
```

Comment:
we can use `vertical` instead of `top` & `bottom`

Reply:
done